### DISCUSSION TITLE
When is next release?

### DISCUSSION THREAD

**Discussion by faraonc** [ory/kratos#3642] [Q&A] [answered]
The last release of 1.0.0 was from Jul 12. There is a bunch of fixes that are AMAZING!

You folks rock!

**Comment by Sytten** [comment]
I would also like to know :)

**Comment by trautonen** [comment]
Especially https://github.com/ory/kratos/pull/3569 would be a desired fix in the release.

**Comment by WoodyWoodsta** [comment]
+1 for a new release here. https://github.com/ory/kratos/discussions/3689 is something I thought was available in v1.0.0 but it appears not!

Edit: I'm also interested in what the release policy is for Ory open-source alongside Ory Network. Is there something somewhere that details what the policy is?

**Comment by vinckr** [comment, marked as answer]
Hey everyone,
Just a quick heads up – please have some more patience.
Looks like the next release won't be until March this year.
No specific date locked in yet.

Currently, we're concentrating a bit more on customers on Ory Network, but don't worry, we haven't dropped off the radar here! 

(In the meantime you can also compile the source directly from master, but I can't not recommend this for production, as it is not tested for all scenarios from our side)

Thanks for your understanding!

**Comment by ysomad** [comment]
Is there any chance #3631 will be implemented in the nearest oss updates?

**Reply by vinckr** [reply]
@ysomad 

There is always a chance :crossed_fingers: ! 
It can for example be contributed by one of the many people interested in the feature.

However we have to pay our maintainers and developers to write code, tests, and also to do code reviews (which can be more work than writing code as you might have experienced yourself) - so prioritization of features is according to commercial interest. AFAICT there is no work going on that particular feature at the moment.

If you are an Ory customer we can allocate people to work on features. 
If you are not an Ory customer, you can still benefit from the work we do by using our software here via GitHub but we can't guarantee that something will be worked on or shipped / released at a certain date. 
Feel free to check out website for what we offer to customers or [contact us directly](https://www.ory.sh/talk-to-us).

I hope this makes it clearer!
